howto/stream/exchange-oob-data.md

```diff
@@ -259,52 +291,72 @@ public class DataReadTask extends AsyncTask<Void, Void, Void> {
 }
 ```
 
-#### Send data to the Anbox WebRTC platform
+#### Send data to the Anbox runtime
 
-To send data to the Anbox WebRTC platform through the file descriptor:
+To send data to the Anbox runtime platform through the file descriptor:
 
 ```
-OutputStream ostream = new FileOutputStream(mFd.getFileDescriptor());
 try {
+    OutputStream ostream = new FileOutputStream(mFd.getFileDescriptor());
     ostream.write(data.getBytes(), 0, data.length());
 } catch (IOException ex) {
     Log.i(TAG, "Failed to write data: " + ex.getMessage());
     ex.printStackTrace();
 }
 ```
 
+<!-- wokeignore:rule=master -->
+For a complete Android example, see the [out_of_band_v2](https://github.com/canonical/anbox-streaming-sdk/tree/master/examples/android/out_of_band_v2) project.
```

Comment:
Shouldn't we tell users to use this right away and then only provide a run through the relevant parts of it? Currently we present it as another option but I think it is unrealistic to think of users writing a new application just to play with the OOB feature. Letting them start with a ready made and working example should be much better. Wdyt?